package.json

```diff
@@ -23,11 +23,10 @@
   },
   "scripts": {
     "build": "gulp",
-    "pretest": "npm run lint",
-    "test": "gulp test",
-    "html-report": "nyc report --temp-directory=./coverage --reporter=html",
-    "jest": "jest --coverage",
-    "coveralls": "nyc report --temp-directory=./coverage --reporter=text-lcov | coveralls",
+    "pretest": "npm run lint && NODE_ENV=test npm run prepublish",
+    "test": "NODE_PATH=test/lib jest test/unit/*.test.js test/isolated/*.test.js test/integration/**/test/*.test.js --coverage",
+    "testonly": "NODE_PATH=test/lib jest test/unit/*.test.js test/isolated/*.test.js test/integration/**/test/*.test.js",
+    "coveralls": "nyc --instrument=false --source-map=false report --temp-directory=./coverage --reporter=text-lcov | coveralls",
```

Comment:
we use `codecov` for this on Jest itself and don't really need nyc in this case.